File: karthikeyan.2512/Java/FlowCharts/Fcp033.java
import java.util.Scanner;
public class Fcp033{
public static void main(String[]args){
Scanner kk=new Scanner(System.in);
int a=kk.nextInt();
Palindrome(a);
}

static void Palindrome(int x){
   int z=x;
   int r,rev=0;
   for(int i=x; x>0;){
    r=x%10;
    rev =(rev*10) + r;
    x /= 10;
}
if(z==rev){System.out.println("yes");}
else{System.out.println("no");}
}
}

File: Aswin.2539/Python/Python_Flowchart_code/FCP033.py
n = int(input())
reverse_digits = 0
palindrome = n
while n > 0:
    digit =  n % 10
    reverse_digits = reverse_digits * 10 + digit
    n = n // 10
if palindrome == reverse_digits:
    print("Yes")
else:
    print("No")